integration/tests/handshake.test.ts

```diff
@@ -885,3 +886,524 @@ test.describe('Client handshake @generic', () => {
     expect(res.status).toBe(200);
   });
 });
+
+test.describe('Client handshake with organization activation @nextjs', () => {
```

Comment:
Scrutiny here: These tests take almost exactly one minute to run on my machine, in parallel with seven worker (via `npm run test:integration:nextjs -- handshake.test.ts`). For comparison, the above generic tests take 32 seconds on my machine.

If we think that's unacceptable, I can do an optimization pass. The lowest-handing-fruit that I see is that each test is starting it's own application server to control it's middleware config. It's likely possible to make the tests serial, overwrite the middleware config per-test, and reload the application.

Reply:
Oh good idea! Since I wrote this initially, @LauraBeatris landed https://github.com/clerk/javascript/pull/4160, which I think I could use to pull middleware options out of headers and avoid starting a new app every time. I'll give that a shot.

Reply:
Done: 3551d2f331c1620a6c2994fc93b52753a2a10c28

It's odd to me though that this hasn't significantly reduced load or test time on my machine. It's still 56 seconds 🤔 .